HW04.Task1/HW04.Task1/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace HW04.Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            for (int bigletters = 90; bigletters <= 90 && bigletters >= 65; bigletters--)
```

Comment:
what a magic numbers - 90, 65? You need to specify the constants or local variables for these numbers